### PR TITLE
Change background color gradient overlay

### PR DESCRIPTION
**Pull request by amyhanv**
fuck

### CODE REVIEW

File: scss/main.scss [hunk 1]
```diff
@@ -23,7 +23,8 @@ body {
 }
 
 .home {
-  background: linear-gradient($dk-btn-color, $lt-btn-color);
+  background: linear-gradient(#000000, #021338);
```
Comment (FlyteWizard) [inline scss/main.scss]: Can we add these colours to the branding.scss and use the variable names here please 💃

**Review by k-erby — DISMISSED** [review]
Like, approved but watch your ~fucking~ language. ❌

**Review by FlyteWizard — CHANGES_REQUESTED** [review]

File: scss/main.scss (hunk 1 above)
Comment (k-erby) [reply scss/main.scss]: yeeeeees YEEEES

Comment (amyhanv) [reply scss/main.scss]: Do we need to? it just random colors and wont be used again

Comment (FlyteWizard) [reply scss/main.scss]: Yeee boi :D Gracias <3

Comment (k-erby) [reply scss/main.scss]: In `branding.scss` assign these to variables like:

```
// Gradient Colors
$black: #000000;
$dark-blue: ##021338;
```

Or something like that. branding is imported in the main css/scss files, so do a similar call like the one you took out!

Comment (amyhanv) [reply scss/main.scss]: okay mams

Comment (k-erby) [reply scss/main.scss]: Yeah, I think it's good to add them to `branding.scss`. It'll be more consistent that way with how we're doing it and if we want to use the colors elsewhere they'll just be in that one file.

Comment (amyhanv) [reply scss/main.scss]: i did it. it is in branding now

**Review by k-erby — APPROVED** [review]
NOICE. 🥇

**Review by FlyteWizard — APPROVED** [review]

**Pull request merged**